reflect.go

```diff
@@ -529,6 +529,9 @@ func (t *Type) arrayKeywords(tags []string) {
 }
 
 func (t *Type) extraKeywords(tags []string) {
+	if t.Extras == nil {
```

Comment:
Any reason this was moved?

Reply:
No reason.  moved to back to setExtras()